code/modules/clothing/clothing.dm

```diff
@@ -529,7 +529,7 @@ BLIND     // can't see anything
 	var/can_adjust = 1
 	var/adjusted = NORMAL_STYLE
 	var/alt_covers_chest = 0 // for adjusted/rolled-down jumpsuits, 0 = exposes chest and arms, 1 = exposes arms only
-	var/obj/item/clothing/tie/hastie = null
+	var/obj/item/clothing/accessory/hasaccessory = null

```

Comment:
Mind removing this `null` here? It's not needed. Also, totally personal preference and it's not enforced anywhere, but I prefer vars like `has_accessory`, a bit easier to read than the entire thing together.